mysql-handler/main.go

```diff
@@ -80,6 +80,10 @@ func Handler(request events.CloudwatchLogsEvent) (Response, error) {
 			for _, field := range common.GetFilterFields() {
 				delete(fields, field)
 			}
+			// Add CloudWatch event metadata
+			hnyEvent.AddField("cw_log_group", data.LogGroup)
```

Comment:
I like it, though for consistency, could we use the `aws.cloudwatch.*` pattern for the column names, like here?

https://github.com/honeycombio/agentless-integrations-for-aws/blob/master/cloudwatch-handler/main.go#L64

Reply:
Maybe this should be pulled into the `common` package so the handlers use the same fields going forward.

Reply:
Renamed the fields as suggested :+1: